charts/tempo-distributed/README.md

```diff
@@ -728,6 +728,7 @@ The memcached default args are removed and should be provided manually. The sett
 | serviceAccount.name | string | `nil` | The name of the ServiceAccount to use. If not set and create is true, a name is generated using the fullname template |
 | storage.admin.backend | string | `"filesystem"` | The supported storage backends are gcs, s3 and azure, as specified in https://grafana.com/docs/enterprise-traces/latest/config/reference/#admin_client_config |
 | storage.trace.backend | string | `"local"` | The supported storage backends are gcs, s3 and azure, as specified in https://grafana.com/docs/tempo/latest/configuration/#storage |
+| storage.trace.block.attribute_columns | list | `[]` | Lis with dedicated attribute columns (only for vParquet3 or later) |
```

Comment:
Could we stick to the name `dedicated_columns`?

Reply:
Changed